components/governance/org.wso2.carbon.governance.api/src/main/java/org/wso2/carbon/governance/api/util/GovernanceUtils.java

```diff
@@ -1857,8 +1863,14 @@ public static void setDefaultLifeCycle(String path, String aspect, Registry regi
                         }
                         if(!subParts[0].equals("name")) {
                             possibleProperties.put(subParts[0], value);
+                            fields.put("overview_" + subParts[0], value.toLowerCase());
+                        } else {
+                            if (artifactConfiguration != null) {
+                                fields.put(artifactConfiguration.getArtifactNameAttribute(), value.toLowerCase());
+                            } else {
+                                fields.put("overview_" + subParts[0], value.toLowerCase());
+                            }
```

Comment:
make it two constants "overview" and "_", may be you can find them in common constants